index.html

```diff
@@ -4,9 +4,51 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles/index.css" rel="stylesheet" />
     <title>Weather Report</title>
 </head>
 <body>
+    <html>
+    <body>
+        <section class="container">
+            <header id="navigation">
+            <h1> Weather Report</h1>
+            <h6> We are in the city of <span id="currentCity"> Seattle </span> 📍</h6>
```

Comment:
Since you have a `h1` before this `h6`, then your next header tag should be `h2` since there is a hierarchy for headers. If you want to make the header with the text "We are in the city of Seattle" smaller, then you can style it with css.